package/fusemodule/package

```diff
@@ -0,0 +1,73 @@
+e!/usr/bin/env bash
+# Copyright (c) 2020 The Toltec Contributors
+# SPDX-License-Identifier: MIT
+
+pkgnames=(fusemodule)
+pkgdesc="FUSE (Filesystem in Userspace) Kernel Module"
+archs=(rm1)
```

Comment:
Consider adding rm2 as a second arch but using logic [similar to the display package](https://github.com/toltec-dev/toltec/blob/3f12d75d214a3e85654e523c285d172e514ddb19/package/display/package#L54-L62) to avoid installing anything for rm2.

Reply:
It's done :) Why is this better?